packages/oasis-actions/src/strategies/ajna/open.ts

```diff
@@ -0,0 +1,78 @@
+import BigNumber from 'bignumber.js'
+import * as ethers from 'ethers'
+
+import ajnaProxyActionsAbi from '../../abi/ajna/ajnaProxyActions.json'
+import { AjnaPosition } from '../../helpers/ajna'
+import { Address, Strategy } from '../../types/common'
+import * as views from '../../views'
+
+export interface OpenArgs {
+  poolAddress: Address
+  dpmProxyAddress: Address
+  quoteAmount: BigNumber
```

Comment:
Make an interface for a `token` with an amount, precision, address, and symbol.

Reply:
Let's do it as an improvement in next step. I don't want to change to interfaces right now it's just POC